### ISSUE TITLE
数据集

### ISSUE THREAD

**Issue by houyuchao** [Alibaba-NLP/MANNER#3]
![image](https://github.com/user-attachments/assets/9d908d8b-e36f-462c-887c-cb8192692d0d)
nin'l您好论文中的这个实验，是以哪个为训练集哪个为测试集呢？？哪个为验证集呢？我看论文说是For fair comparison, we
use the same sampled episodes and dataset splits as
in (Hou et al., 2020), where two of the four datasets
are used for training, one for validation and the
other for test. For example, to evaluate the performance on Ontonotes, we take WNUT and GUM as
the training sets and CoNLL as the validation set.两个数据集当作训练集吗？？？
